### PR TITLE
feat: add Microsoft Active Accessibility (MSAA)

### PR DESCRIPTION
**Pull request by Fydon**
I've used [WinLister](https://www.nirsoft.net/utils/winlister.html) to find the class of the windows

- [x] I have formatted `applications.yaml` with `komorebic fmt-asc`.

### CODE REVIEW

File: applications.yaml [hunk 1]
```diff
@@ -173,6 +173,17 @@
...
+  identifier:
+    kind: Exe
+    id: dbeaver.exe
+    matching_strategy: Equals
+  float_identifiers:
+  - kind: Class
+    id: '#32770'
+    comment: All popups appear to use this
+  options:
+  - border_overflow
```
Comment (sitiom) [inline applications.yaml:186]: DBeaver window does not seem to border overflow for me:
image

**Review by sitiom — CHANGES_REQUESTED** [review]

File: applications.yaml (hunk 1 above)
Comment (Fydon) [reply applications.yaml]: I have `active_window_border` set to true. Yesterday that border was being overlapped by the app. Today on the first load it overlapped, but then when I moved the window to make it smaller for a screenshot, it corrected the border. I tried loading the app a few times afterwards and the border didn't overlap. There is probably a different issue I'm experiencing. I've removed that option.

File: applications.yaml [hunk 2]
```diff
@@ -173,6 +173,15 @@
...
   - border_overflow
   - tray_and_multi_window
+- name: DBeaver
+  identifier:
+    kind: Exe
+    id: dbeaver.exe
+    matching_strategy: Equals
+  float_identifiers:
+  - kind: Class
+    id: '#32770'
```
Comment (sitiom) [inline applications.yaml:183]: For reference: https://learn.microsoft.com/en-us/windows/win32/winauto/dialog-box

>A dialog box is a temporary window that an application creates to retrieve user input. An application uses dialog boxes to prompt the user for additional information about commands that the user has chosen from a menu. A dialog box contains one or more controls (child windows) with which the user enters text, chooses options, or directs the action of the command.
>
>The window class name for dialog boxes is "#32770".

Perhaps this should be applied as a standalone rule.

**Review by sitiom — CHANGES_REQUESTED** [review]

File: applications.yaml (hunk 2 above)
Comment (Fydon) [reply applications.yaml:179]: Thank you for your continued help with my proposed changes. I think I've updated the changes as you've requested. I've removed the application identifier for DBeaver as there is nothing specific to the application left.

**Review by sitiom — APPROVED** [review]
Thanks!

Note: added empty identifier to please komorebi

**Comment by Fydon** [comment]
Yes. However it seems from the [link sitiom found](https://learn.microsoft.com/en-us/windows/win32/winauto/dialog-box) that this class is common for dialog boxes from various applications.

image

**Comment by LGUG2Z** [comment]
Aha I see, thank you! I'll think about how to update the schema for entries like this in the future, but for now we can keep adding float identifiers like this as we are with the empty class 👌

**Pull request merged**